pages/[slug]/smsmfa/index.tsx

```diff
@@ -1,18 +1,29 @@
-import {SMSForm} from "@/components/SMSForm";
+import {SMSSendForm} from "@/components/SMSSendForm";
 import {useRouter} from "next/router";
+import {SMSAuthenticateForm} from "@/components/SMSAuthenticateForm";
 
 const App = () => {
   const router = useRouter();
   const orgID = router.query.org_id as string;
   const memberID = router.query.member_id as string;
   const sent = router.query.sent as string;
 
+  if(sent === "true") {
```

Comment:
fun tidbit/aside - an alternative way of writing this could be
```tsx
const Component = sent === "true" ? SMSAuthenticateForm : SMSSendForm;
return (
  <div className="card">
    <Component
      orgID={orgID}
      memberID={memberID}
    />
  </div>
)
```

since components are just functions

Reply:
that was a fun one https://github.com/stytchauth/stytch-b2b-nextjs-example/pull/20/commits/9ef1a7368fdd456d0c6c5078da564782be60e7ac